ksql-rest-app/src/test/java/io/confluent/ksql/api/auth/BasicCallbackHandlerTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.auth;
```

Comment:
Personally I am not a big fan of these kinds of fine grained unit tests for simple classes. I don't think they tell you much, plus they're a fair amount of work to write ;)

Reply:
That's fair. The value I see in simple tests like these is that I often find it easier to understand what a class is meant to do by scanning the unit tests rather than reading the code directly, but perhaps it was overkill this time. I'll leave them in since I've already written them, but if they become problematic later (if we, e.g., evolve the class in a way that requires updating the tests) I'm fine with removing them.